### ISSUE TITLE
More documentation, please?

### ISSUE THREAD

**Issue by divs1210** [mlanza/cloe#1]
Could you add more documentation or example code to illustrate how this lib can be used?

Saw this project on a thread in r/Clojure, and it looks really nice!

Simple examples of core features would be nice, like reactives, protocols, etc.

**Comment by mlanza** [comment]
I will put out some examples in the near future.  Stay tuned.

I've been using this library for years now, privately, and never really released it.  I was trying to avoid the open source maintenance shackles.  Plus, I've made drastic changes over the years, never worrying I'd harm others.

I think my code is descent.  It certainly works!  But this was my personal way of using Rich Hickey's ideas online, without transpilation.  In a way, I guess I hoped someone might see it and be inspired to take things to the next level.

**Comment by mlanza** [comment]
In the meantime, here are some examples to investigate:

https://github.com/mlanza/todo
https://github.com/mlanza/treasure-quest
https://github.com/mlanza/pickomino (not totally complete, but close enough)

I have larger projects which I haven't exposed to the public, but which would themselves require more docs than exist.

**Comment by divs1210** [comment]
Thank you! Will have a look when I can!

**Comment by mlanza** [comment]
I added a bit more to the main README.

**Comment by mlanza** [comment]
I have refined the main README again.  It now cites the 3 examples from above.  None are too big and each is sufficient enough in itself to help someone see how to develop an app.  The big concept is starting with a functional core (pure code) and finishing with an imperative shell (impure ui).

I will gradually refine the docs.  Understand that I created this library for my personal and professional use and that I haven't followed the typical open source path.  So while it's mostly invisible to a wider audience that's not saying it isn't good or that it's not prod ready.  It's been used in prod for years.  I'm just not taking enhancement requests.  

Consider Clojure itself.  After a few of its big developments like `core.async` and `spec` it's relatively stable.  But the fact it's not continually adding newness doesn't means it's not good.

I'm just one person with a family.  I peeked in on [underscore](https://github.com/jashkenas/underscore/issues) and [lodash](https://github.com/lodash/lodash/issues) over the years (both of which I used to use) and those guys have been swamped with endless requests and those are just small libraries by comparison.  I could never keep up with that kind of activity.

That said, understand that gradually tailoring this for my own use has been great.  I'm not stressed by the ever-changing technological landscape.  I mention this only to say if you managed your own fork you'd be essentially doing what I've been happily doing for nearly a decade.

I will support fixes and optimizations, but at this point I'm not equipped to handle lots of change and ensure safety for everyone downstream.  I'd need a team for that.
